src/utils/SignerUtils.ts

```diff
@@ -1,42 +1,103 @@
-import { Wallet, retrieveGckmsKeys, getGckmsConfig } from "./";
-// const args = require("minimist")(process.argv.slice(2));
+import { Wallet, retrieveGckmsKeys, getGckmsConfig, isDefined } from "./";
 
-import minimist from "minimist";
-const args = minimist(process.argv.slice(2));
+/**
+ * Signer options for the getSigner function.
+ */
+export type SignerOptions = {
+  /*
+   * The type of wallet to use.
+   * @note If using a GCKMS wallet, the gckmsKeys parameter must be set.
+   */
+  keyType: string;
```

Comment:
Since we only support 3 types, is it worth locking this down?

```suggestion
  keyType: "gckms" | "mnemonic" | "privateKey";
```

Reply:
Done: https://github.com/across-protocol/relayer-v2/pull/888/commits/41616e22cdc16c85e2669be3eafa04dc3889e8a4